src/components/DpAutocomplete/DpAutocomplete.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+
+import DpAutocomplete from './DpAutocomplete.vue'
+
+const meta: Meta<typeof DpAutocomplete> = {
+    component: DpAutocomplete,
+    title: "Components/DpAutocomplete",
+    argTypes: {
+        'searchChanged': { action: 'searchChanged' }
+    }
+}
+
+export default meta
+type Story = StoryObj<typeof DpAutocomplete>
+
+export const Default: Story = {
+    args: {
+        height: '28px',
+        label: 'label',
```

Comment:
we don't need to pass the default values.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/396/commits/c6bc9c28795611cfd6215e05c3ef69b2406b3649